cloud_integrations_add_kubernetes.md

```diff
@@ -0,0 +1,7 @@
+### Adding a Kubernetes Cluster
+
+Adding a Kubernetes cluster for monitoring requires deploying a **Wavefront proxy** and **Wavefront collector** inside your Kubernetes cluster.
+
+You can follow the step-by-step guided Helm instructions if applicable, or follow the manual instructions to deploy the required components yourself.
```

Comment:
You have two choices: 
* Follow the simple Helm install steps on the left. 
* Perform manual installation ( ??link??).